### PR TITLE
Fix security task instructions.

### PR DESCRIPTION
**Pull request by myidpt**
Fix more places in security tasks, including invalid code references and cleanup instructions.

### CODE REVIEW

File: content/docs/tasks/security/health-check.md [hunk 1]
```diff
@@ -41,7 +42,7 @@ this feature is not needed if the production setup is not using the
 Deploy Citadel with health checking enabled.
 
 ```command
-$ kubectl apply -f @install/kubernetes/istio-citadel-with-health-check.yaml@
+$ kubectl apply -f install/kubernetes/istio-citadel-with-health-check.yaml
```
Comment (geeknoid) [inline content/docs/tasks/security/health-check.md]: Please add back the @@ symbols. Those render as links to that file on GitHub. See https://istio.io/about/contribute/writing-a-new-topic#showing-references-to-istio-github-files

File: content/docs/tasks/security/health-check.md [hunk 2]
```diff
@@ -126,18 +127,10 @@ continuously failed health checks.
...
-    $ kubectl delete svc istio-citadel -n istio-system
-    ```
-
-*   To remove Citadel:
+*   To disable health checking on Citadel (with global mutual TLS enabled):
 
     ```command
-    $ kubectl delete -f @install/kubernetes/istio-citadel-with-health-check.yaml@
-    $ kubectl delete svc istio-citadel -n istio-system
+    $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
```
Comment (geeknoid) [inline content/docs/tasks/security/health-check.md]: Add @@ symbols.

File: content/docs/tasks/security/plugin-ca-cert.md [hunk 3]
```diff
@@ -52,7 +53,7 @@ The following steps enable plugging in the certificates and key into Citadel:
 1.  Redeploy Citadel, which reads the certificates and key from the secret-mount files:
 
     ```command
-    $ kubectl apply -f @install/kubernetes/istio-citadel-plugin-certs.yaml@
+    $ kubectl apply -f install/kubernetes/istio-citadel-plugin-certs.yaml
```
Comment (geeknoid) [inline content/docs/tasks/security/plugin-ca-cert.md]: @@

File: content/docs/tasks/security/plugin-ca-cert.md [hunk 4]
```diff
@@ -133,10 +134,10 @@ This requires you have `openssl` installed on your machine.
     $ kubectl delete secret cacerts -n istio-system
     ```
 
-*   To remove the Istio components:
+*   To switch to Citadel using self-signed certificate (with global mutual TLS enabled):
 
     ```command
-    $ kubectl delete -f @install/kubernetes/istio-auth.yaml@
+    $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
```
Comment (geeknoid) [inline content/docs/tasks/security/plugin-ca-cert.md]: @@

Comment (gyliu513) [inline content/docs/tasks/security/plugin-ca-cert.md]: Why cleanup create istio again?

File: content/docs/tasks/security/mutual-tls.md [hunk 5]
```diff
@@ -20,6 +20,7 @@ This task assumes you have a Kubernetes cluster:
     $ kubectl apply -f @install/kubernetes/istio-auth.yaml@
     ```
     _**OR**_
+
```
Comment (gyliu513) [inline content/docs/tasks/security/mutual-tls.md]: this is not needed.

**Comment by geeknoid** [comment]
Do we want to proceed with this PR or should it be closed?

File: content/docs/tasks/security/health-check.md (hunk 1 above)
Comment (sdake) [reply content/docs/tasks/security/health-check.md]: @geeknoid those files are not part of the distribution.

File: content/docs/tasks/security/plugin-ca-cert.md (hunk 3 above)
Comment (sdake) [reply content/docs/tasks/security/plugin-ca-cert.md]: this file is not part of the distribution.

**Comment by geeknoid** [comment]
Please reopen if needed.

**Pull request closed without merging**